### PR TITLE
Add Limitations Warning of Attendance Report API

### PR DESCRIPTION
**Pull request by yuyaolian-msft**

### CODE REVIEW

File: api-reference/beta/api/meetingattendancereport-get.md [hunk 1]
```diff
@@ -56,6 +56,7 @@ GET /users/{userId}/onlineMeetings/{meetingId}/meetingAttendanceReport
 >
 >- The `/meetingAttendanceReport` path is deprecated. Going forward, use the `/attendanceReports` path to retrieve attendance reports for an online meeting.
 >- The `/meetingAttendanceReport` path will remain in beta for backward compatibility. However, to get the same response, you need to add the `expand` query option. For details, see the [Optional query parameters](#optional-query-parameters) section.
+>- The **attendanceRecords** currently does not contain the **attendanceInterval** from a breakout room.
```
Comment (frankpeng7) [inline api-reference/beta/api/meetingattendancereport-get.md:59]: The **attendanceRecords** property

Comment (satyakonmsft) [inline api-reference/beta/api/meetingattendancereport-get.md:59]: should we just say a generic stmt like "**attendanceRecords** currently does not support returning information about a breakout room"

wondering if saying "does not contain **attendanceInterval** from a breakout room" implies some other info from a breakout room is still available.

File: api-reference/v1.0/api/meetingattendancereport-get.md [hunk 2]
```diff
@@ -45,6 +45,10 @@ GET /users/{userId}/onlineMeetings/{meetingId}/attendanceReports/{reportId}
 >- `meetingId` is the **id** of an [onlineMeeting](../resources/onlinemeeting.md) object.
 >- `reportId` is the **id** of an [meetingAttendanceReport](../resources/meetingAttendanceReport.md) object.
 
+> [!CAUTION]
+>
+>- The **attendanceRecords** currently does not contain the **attendanceInterval** from a breakout room.
```
Comment (satyakonmsft) [inline api-reference/v1.0/api/meetingattendancereport-get.md:50]: should we just say a generic stmt like "**attendanceRecords** currently does not support returning information about a breakout room"

wondering if saying "does not contain **attendanceInterval** from a breakout room" implies some other info from a breakout room is still available.

File: api-reference/beta/api/meetingattendancereport-get.md (hunk 1 above)
Comment (yuyaolian-msft) [reply api-reference/beta/api/meetingattendancereport-get.md:59]: thank you!

**Review by satyakonmsft — APPROVED** [review]

**Review by isvargasmsft — APPROVED** [review]

**Pull request merged**